shaper/libs/parser.py

```diff
@@ -260,6 +267,12 @@ def write(self, data, path):
 
 class PropertyParser(TextParser):
 
+    @staticmethod
+    def __process_multiline_string(string):
```

Comment:
pls don't use the double underscore, it is something like overhead and java-like style) a single underscore would be fair enough to understand that this method should be used only within the `PropertyParser` class body

Reply:
done